packages/cli/src/commands/link/getPlatformName.ts

```diff
@@ -0,0 +1,7 @@
+type names = {[key: string]: string};
+const names: names = {
```

Comment:
Having the type with the exact same name as the variable is a bit confusing. Maybe inline it instead?:

```suggestion
const names: {[key: string]: string} = {
```

Reply:
yeah ...this suggestion looks neat ! 👍

Reply:
done